content/en/blog/_posts/2020-12-02-dockershim-faq.md

```diff
@@ -3,7 +3,6 @@ layout: blog
 title: "Dockershim Deprecation FAQ"
 date: 2020-12-02
 slug: dockershim-faq
-aliases: [ '/dockershim' ]
 ---
 
 This document goes over some frequently asked questions regarding the Dockershim
```

Comment:
```suggestion

_**Update**: There is a [newer version](/blog/2022/02/16/dockershim-faq/) of this article available._

This document goes over some frequently asked questions regarding the Dockershim
```

Reply:
Did you want a redirect instead, @sftim? I don't want that alias going to a marked outdated page.

Reply:
If we do a second update to this article, we should edit all previous incarnations. I'm sanguine about the risk of someone forgetting to do that.